src/pubsub.c

```diff
@@ -532,6 +532,15 @@ int pubsubPublishMessage(robj *channel, robj *message, int sharded) {
     return pubsubPublishMessageInternal(channel, message, sharded ? pubSubShardType : pubSubType);
 }
 
+/* Publish messages to all the subscribers. */
+int pubsubPublishMessages(robj *channel, robj **messages, int count, int sharded) {
```

Comment:
the function name is too close to its singular counterpart so it hurts readability. can we consider the following instead?
```suggestion
int pubsubPublishMultiMessages(robj *channel, robj **messages, int count, int sharded) {
```